build.sh

```diff
@@ -152,7 +152,7 @@ if (( ${NUMARGS} == 0 )) || hasArg libcudf; then
           -DCMAKE_CXX11_ABI=ON \
           ${GPU_ARCH} \
           -DBUILD_BENCHMARKS=${BENCHMARKS} \
-          -DCMAKE_BUILD_TYPE=${BUILD_TYPE} ..
+          -DCMAKE_BUILD_TYPE=DEBUG ..
```

Comment:
What's this change for? This can script can already be invoked with `-g` set `BUILD_TYPE` to DEBUG.

Reply:
@raydouglass This was a goof I overlooked and should not have committed. Long story but was something I needed for a very specific environment I was testing on. I will change it back.